src/api/spec/support/vcr.rb

```diff
@@ -6,21 +6,23 @@
   config.default_cassette_options = { record: :once }
   config.allow_http_connections_when_no_cassette = true
   config.configure_rspec_metadata!
+  config.debug_logger = File.open(Rails.root.join('log', 'vcr.log'), 'w')
```

Comment:
Why?

Reply:
Because I can :)

I didn't find an easier (as in directly blobbing into test.log) way to find which URLs are actually fetched by tests.

Reply:
Sure to debug but every time we run tests?

Reply:
Leave it as comment then?

Reply:
Sure